storyblok/storyblok-design-system
# feat(work-360): introduce scoped slot header to sb datatable column

## Pull request type

Jira Link: [WORK-360](https://storyblok.atlassian.net/browse/WORK-360)




## How to test this PR
1. Test `SbDataTable` new `header` slot to make sure the content is being properly displayed on the respective headers
2. Test `SbDataTable` without the new `header` slot to make sure it's working properly (as before)



## What is the new behavior?



- Introduce a new `header` slot that allows the developer to set a customized slot for a given column

## Other information
